src/themes/withStyles.js

```diff
@@ -0,0 +1,21 @@
+import globalCache from 'global-cache';
+import aphroditeInterface from 'react-with-styles-interface-aphrodite';
+import ReactWithStylesThemedStyleSheet from 'react-with-styles/lib/ThemedStyleSheet';
+import { css, withStyles, ThemeProvider } from 'react-with-styles';
+
+import DefaultTheme from './default';
+
+const ThemedStyleSheet = globalCache.setIfMissingThenGet(
+  'rheostat withStyles',
+  () => {
+    // Registering the default theme more than once clobbers all of the
+    // previously created styles, which causes errors to happen. This can happen
+    // if this file appears in more than one bundle on the same page. To avoid
+    // this, we are using a global cache.
+    ReactWithStylesThemedStyleSheet.registerDefaultTheme(DefaultTheme);
+    ReactWithStylesThemedStyleSheet.registerInterface(aphroditeInterface);
+    return ReactWithStylesThemedStyleSheet;
+  }
+);
+
+export { css, withStyles, ThemeProvider, ThemedStyleSheet };
```

Comment:
I think instead of doing all of this here, we will need to provide instructions for folks about how to wire this up properly in their own project. We probably need to keep something like this around for storybook, but the component shouldn't depend on this because it will couple it with Aphrodite.